index.html

```diff
@@ -18,23 +18,147 @@
       </nav>
     </div>
   </header>  
```

Comment:
- [OPTIONAL] Please kindly consider fixing the margins on your page, The page is not well centralized. See images below. 
***Your page at width: 400px***

![lucas01](https://user-images.githubusercontent.com/65852621/138535214-f1a621a8-6da9-468d-ba94-9d661a50f1b4.png)

![lucas02](https://user-images.githubusercontent.com/65852621/138535220-c5a26fc7-6ee2-4633-870a-6615d35fce4f.png)

![lucas03](https://user-images.githubusercontent.com/65852621/138535226-f0929583-a0e7-4831-a375-12690f0b1479.png)

![lucas04](https://user-images.githubusercontent.com/65852621/138535234-36ebb505-68de-4c62-b19e-cff73d208a50.png)

![lucas05](https://user-images.githubusercontent.com/65852621/138535244-c79f4b01-34de-4aca-9710-594c9a910a53.png)

***The [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)***